go/libraries/doltcore/sqle/dtables/conflicts_tables_prolly.go

```diff
@@ -240,10 +242,22 @@ func (itr *prollyConflictRowIter) putConflictRowVals(ctx *sql.Context, c conf, r
 			r[itr.t+itr.kd.Count()+i] = f
 		}
 	}
+	r[itr.t+itr.kd.Count()+itr.theirsVD.Count()] = getDiffType(c.bV, c.tV)
 
 	return nil
 }
 
+func getDiffType(base val.Tuple, other val.Tuple) string {
+	if base == nil {
+		return "added"
```

Comment:
Aren't there constants for these strings defined somewhere? There should be, i would think either in the merge package or the doltdb package